packages/datakit-bridge-github/datakit-bridge-github.0.10.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer:   "thomas@gazagnaire.org"
+authors:      ["Thomas Leonard" "Magnus Skjegstad"
+               "David Scott" "Thomas Gazagnaire"]
+license:      "Apache"
+homepage:     "https://github.com/moby/datakit"
+bug-reports:  "https://github.com/moby/datakit/issues"
+dev-repo:     "https://github.com/moby/datakit.git"
+doc:          "https://docker.github.io/datakit/"
+
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+build-test: ["jbuilder" "runtest" "tests/%{name}%"]
+
+depends: [
+  "jbuilder" {build}
+  "cmdliner"
+  "lwt" {>= "2.7.1"}
+  "datakit-github" {>= "0.10.0"}
+  "logs" "fmt" "mtime" "asl" "win-eventlog"
+  "uri" {>= "1.8.0"}
+  "hvsock" {>= "0.8.1"}
+  "hex" "nocrypto" "conduit"
+  "prometheus-app"
+  "protocol-9p" {>= "0.8.0"}
+  "datakit-client" {>= "0.10.0"}
+  "github-hooks" {>= "0.1.1"}
+  "github" {>= "2.1.0"}
+  "datakit" {test & >= "0.10.0"}
+]
```

Comment:
[There](https://travis-ci.org/ocaml/opam-repository/jobs/226867823)'s a missing dependency of some sort:

```
#=== ERROR while installing datakit-bridge-github.0.10.0 ======================#
[...]
# Error: Findlib package "datakit-server" not found.
# -> required by "tests/common/jbuild (context default)"
# Hint: try: jbuilder external-lib-deps --missing -p datakit-bridge-github -j 4
```